src/tyr/actor.cc

```diff
@@ -51,69 +51,103 @@ void actor_t::cleanup() {
   pimpl->cleanup();
 }
 
+std::string actor_t::act(Api& api, const std::function<void()>* interrupt) {
```

Comment:
in pbf mode you dont need to specify the action separately, instead you just set it on the pbf options object and we can use that to tell which action you want to perform. i'll expose this method in the python bindings in a future pr

Reply:
I guess returning the pbf bytes as string despite the passed api object being filled out is not a big deal (on any level), but I'd leave a small comment/todo maybe?

Reply:
instead i added javadoc to the header file that was sorely m issing which spells out exactly everything that is going on in the fuction